executor/common_kvm.h

```diff
@@ -5,6 +5,52 @@
 
 // Common KVM-related definitions.
 
+#include "common_kvm_syzos.h"
+#include "kvm.h"
+
+extern char* __start_guest;
```

Comment:
This should also go into `#if SYZ_EXECUTOR || __NR_syz_kvm_add_vcpu` section, so that it's stripped from reproducers when KVM is not used.